src/component/pages/Categories.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function Categories() {
+  return (
+    <div>Categories</div>
+  );
+}
```

Comment:
- [ ] Following the project requirements, the Categories component should display only a button with the text "Check status".